### ISSUE TITLE
[GENERAL] Jaws of life do not open firelocks (And mech construction)

### ISSUE THREAD

**Issue by tacoguy7765093** [CHOMPStation2/CHOMPStation2#7260]
#### Brief description of the bug
Using a jaws of life on a shut fire lock does not open it, a regular crowbar and prybar still work though.

You are also unable to use the wirecutter step in building mechs with the jaws of life, regular wirecutters work tho.

#### Steps to reproduce
- (Step 1) Find a closed firelock
- (Step 2) Try crowbaring it with a jaws of life.
